src/azul/indexer.py

```diff
@@ -325,6 +325,10 @@ def _aggregate(self, contributions: List[Contribution]) -> List[Aggregate]:
         # Aggregate contributions for the same entity
         aggregates = []
         for entity, contributions in contributions_by_entity.items():
+            log.info('Found %i contributions for entity %s/%s during aggregation',
```

Comment:
This is too late for what I was envisioning but may be useful, so leave it in. I had envisioned to log around [here](https://github.com/DataBiosphere/azul/pull/1579/files#diff-c98e1dc4399542337daecdeafc7da4feL297).

```suggestion
            log.info('Selected %i contributions to be aggregated for entity %s/%s.',
```